### DISCUSSION TITLE
Failed to create erlang.cookie when HOMEDRIVE is a network share

### DISCUSSION THREAD

**Discussion by swank6597** [rabbitmq/rabbitmq-server#8150] [Other]
### Describe the bug

Tried running these commands but they all generated a crash dump (attached) for each command.

```
.\rabbitmq-service.bat stop
.\rabbitmq-service.bat remove
set
.\rabbitmq-service.bat install
.\rabbitmq-service.bat start
```
the tools copy the system erlang cookie to %userprofile% to enable access to the tools. It looks like there is something retargeting that to the Directory. If the Directory is a network share then the local admin is unlikely to be able to access it.

A workaround was found.   RabbitMQ is set to install to %homedrive%%homepath%.   Out of the box those two variables are c:\Users\Username.
If a user changes it at the domain level to a network share, the rabbit install will use that instead.

Is there any solution to keep the erlang cookie path sustainable?

### Reproduction steps

1. Keep HOMEDRIVE as a Network Share
2. Install RabbitMQ on the local machine.
3. Start the RabbitMQ Service

### Expected behavior

RabbitMQ service must start without any error messages

**Comment by michaelklishin** [comment]
If you cannot install RabbitMQ using an administrative user and the user does not have the permissions for HOMEDRIVE, the only option I can think of is to change HOMEDRIVE to a location that it will have the write permissions to.

Perhaps running RabbitMQ in WSL
or a container could help, assuming that CLI tools will be used from the container itself.

See https://rabbitmq.com/windows-quirks.html#cookie-location
